### PR TITLE
Featuredesignteardown

### PR DESCRIPTION
**Pull request by Igors78**
The project is the fifth of the Microverse curriculum HTML/CSS module. 
Project created as a grayscale heat map of the website that indicates which elements have the greatest weight in the visual hierarchy. The goal of this exercise is to start training ourselves to think in terms of visual hierarchy, typography, and design principles.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,571 @@
...
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__topmore" href="#"><i class="fas fa-cat fa-2x"></i></a>
+        <i class="fas fa-search fa-2x zover"></i>
+        <input class="navbar__topsearch" type="text" />
+      </nav>
+    </header>
+    <article class="article1">
```
Comment (umairarshadbutt) [inline index.html:31]: Replace `article` tag with a `section` tag

File: index.html [hunk 2]
```diff
@@ -1,15 +1,571 @@
...
+          <a href="#" class="comment1"></a>
+        </div>
+        <div class="arrow">
+          <a class="white" href="#">
+            <i class="fa fa-long-arrow-right fa-2x" aria-hidden="true"></i>
+          </a>
+        </div>
+      </div>
+    </article>
+    <article class="article2">
```
Comment (umairarshadbutt) [inline index.html:171]: Replace `article` tag with a `section` tag

File: index.html [hunk 3]
```diff
@@ -1,15 +1,571 @@
...
+          <p class="bottomtext-item"></p>
+          <p class="bottomtext-item"></p>
+        </div>
+        <div class="arrow2">
+          <a class="white-arrow" href="#">
+            <i class="fa fa-long-arrow-right fa-2x" aria-hidden="true"></i>
+          </a>
+        </div>
+      </div>
+      <div class="article2-item">
```
Comment (umairarshadbutt) [inline index.html:192]: Replace `div` tag with an `article` tag

File: index.html [hunk 4]
```diff
@@ -1,15 +1,571 @@
...
+        </div>
+        <div class="arrow">
+          <a class="white" href="#">
+            <i class="fa fa-long-arrow-right fa-2x" aria-hidden="true"></i>
+          </a>
+        </div>
+      </div>
+    </article>
+    <article class="article2">
+      <div class="article2-item">
```
Comment (umairarshadbutt) [inline index.html:172]: Replace `div` tag with an `article` tag

File: index.html [hunk 5]
```diff
@@ -1,15 +1,571 @@
...
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__toplinks" href="#"></a>
+        <a class="navbar__topmore" href="#"><i class="fas fa-cat fa-2x"></i></a>
+        <i class="fas fa-search fa-2x zover"></i>
+        <input class="navbar__topsearch" type="text" />
+      </nav>
+    </header>
+    <article class="article1">
+      <div class="article1-right zover">
```
Comment (umairarshadbutt) [inline index.html:32]: Replace `div` tag with an `article` tag

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Igors78 and @KinezaEnaelle
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly and work on the suggested changes  👇.
- [ ]  Navigation bar should use `nav` element with `ul/li` inside
- [ ] Use the article tag properly on a complete landing page. See inline comments
- [ ]  Footer contains centered links. Please update your container in the footer
- [ ] Remove the horizontal scroll bar
scrollbar


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great work the site is looking amazing!

## Required Changes
### [Mandatory] :red_circle: 

- [ ] Leave only production-ready files in the repo, add the un-process ones to .gitignore
image

- [x] Wrapp the sections in a ```<main>``` HTML tag

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

You may now merge to master! 

- [x] Before merging add de .scss file to .gitignore.

## Congratulations team!! See you at the top :rocket: !

**Pull request merged**